devito/symbolics/extended_dtypes.py

```diff
@@ -15,6 +18,45 @@
 }
 
 
+class NoDeclStruct(ct.Structure):
+    # ctypes.Structure that does not generate a struct definition
```

Comment:
Should be docstring

Reply:
Ok, presumably same for the `c_complex` etc.?

Reply:
Yeah, I think so